src/models/transactions/transaction.ts

```diff
@@ -45,3 +63,104 @@ export interface TransactionAndMetadata {
   transaction: Transaction;
   metadata: Metadata;
 }
+
+/**
+ *
+ * @param tx - A Transaction.
+ * @returns
+ * @throws {ValidationError} When the Transaction is malformed.
+ */
```

Comment:
This is mildly incorrect TSdocs formatting.

Reply:
Linter will fix this

Reply:
I know, I just want to resolve linter errors in new code now, so we don't have to deal with it later

Reply:
This has now been changed and done correctly